docker/tests.sh

```diff
@@ -0,0 +1,20 @@
+
+#!/bin/bash
+# Copyright (c) 2018-2019 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+echo "Pulling samples repo.."
+git clone https://github.com/vmware-samples/vmware-blockchain-samples.git
+cd vmware-blockchain-samples/supply-chain
+git checkout mharrison/functional-tests-on-concord
+
+echo "Building Supply Chain Sample"
+docker build -t supply-chain:latest .
+
+echo "Startup concord"
+cd ../../
+docker-compose -f docker/compose/simple4.yml up -d
+sleep 30
```

Comment:
I'm a little afraid the sleep will be unreliable in a CI environment.
Can we wait for the ethRPC node to be up? (maybe use curl in a loop until we get a response?)

Reply:
👍 Sure no problem. 

I'm happy to see these tests pass on first try.